webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx

```diff
@@ -277,9 +277,7 @@ my_model.load_state_dict(ckpt['models_state_dict'][0])`;
             </Dropdown>
           </div>
         </div>
-        <Spinner spinning>
-          <InfoBox rows={infoRows} separator={false} />
-        </Spinner>
+        <InfoBox rows={infoRows} separator={false} />
```

Comment:
infoRows is only dependent on `users` and `modelVersion`. `modelVersion` will always be defined where as `users` is a `Loadable` state, but it already has it's own local spinner. So it's ok to go ahead and always render `Infobox`.